apps/music/js/utils.js

```diff
@@ -92,11 +92,15 @@ function createListElement(option, data, index, highlight) {
     case 'album':
     case 'title':
       var artistBdi;
+      var albumImg;
       // Use background image instead of creating img elements can reduce
       // the amount of total elements in the DOM tree, it can save memory
       // and gecko can render the elements faster as well.
       AlbumArt.getCoverURL(data).then(function(url) {
-        li.style.backgroundImage = 'url(' + url + ')';
+        albumImg = document.createElement('img');
```

Comment:
I think we should create the `<img>` element immediately, not in the callback here. Maybe just put it right before the callback?